cli/command/plugin/testdata/plugin-upgrade-terminate.golden

```diff
@@ -0,0 +1,2 @@
+Upgrading plugin foo/bar from localhost:5000/foo/bar:v0.1.0 to localhost:5000/foo/bar:v1.0.0
+Plugin images do not match, are you sure? [y/N] 
```

Comment:
Missing a newline at the end here? (The other new golden files have them 🤔)

Reply:
@Benehiko could we fix this?

Reply:
Right, this is likely due to us missing a newline when printing things somewhere